code/mob/living/stamina_stuns.dm

```diff
@@ -91,6 +91,36 @@
 	if(src.stamina_bar?.last_update != TIME) src.stamina_bar?.update_value(src)
 	return
 
+/// Adds 'get up' stamina/stun reduction, from taking beatdown damage.
+/mob/proc/get_revenge_stamina(stamina_damage, brute, burn, damage_type )
+	return
+
+/mob/living/get_revenge_stamina(stamina_damage, brute, burn, damage_type )
```

Comment:
This name doesn't really make sense anymore since it's not giving stamina, PR description also needs updating.